src/frontend/src/pages/login/login.tsx

```diff
@@ -1,7 +1,34 @@
 import React from 'react';
+import Grid from '@material-ui/core/Grid';
+
+// for fields
+import TextField from '@material-ui/core/TextField';
+
+// for submit
+import Button from '@material-ui/core/Button';
+import {
+    Link,
+} from "react-router-dom"
 
 export const Login: React.FC = () => {
     return (
-        <div>Login Page</div>
+        <div>
+            <Grid container spacing={3}>
+                <Grid item xs={12}>
```

Comment:
```suggestion
            <Grid container direction="column" justify="center" alignItems="center">
                <Grid item>
```
It's possible to switch the grid direction as per https://material-ui.com/components/grid/#interactive so that you don't need to use any 12 columns per item hacks.